careers/application-process/index.html

```diff
@@ -281,13 +281,17 @@ <h3 class="u-hide-on-mobile">Asking About your Application</h3>
     <div class="block
                 block__padded-top
                 block__border-top
+                block__flush-bottom
                 content-l">
-        <section class="content-l_col content-l_col-1-2">
+        <article class="content-l_col content-l_col-1-3">
+            {% include "_template-current-openings.html" %}
+        </article>
+        <article class="content-l_col content-l_col-1-3">
             {% include "_template-working-at-cfpb.html" %}
-        </section>
-        <section class="content-l_col content-l_col-1-2">
+        </article>
```

Comment:
I'm adding this to the list of gray-area things I've run into to ask at the
WebAIM training.

On Fri, Jul 10, 2015, 2:52 PM Scott Cranfill <notifications@github.com>
wrote:

> In careers/application-process/index.html
> <https://github.com/cfpb/cfgov-refresh/pull/740#discussion_r34384808>:
>
> >              {% include "_template-working-at-cfpb.html" %}
> > -        </section>
> > -        <section class="content-l_col content-l_col-1-2">
> > +        </article>
>
> It does seem a bit of a great area, but I think would come down on
> Kimberly's side, given the arguments presented so far.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfpb/cfgov-refresh/pull/740/files#r34384808>.
>